lib/stack.js

```diff
@@ -10,4 +10,4 @@ var Stack = function Stack() {
   _classCallCheck(this, Stack);
```

Comment:
Consider adding `lib/` to your `.gitignore` to reduce the clutter of autogenerated files in your repo.